README.md

```diff
@@ -1,38 +1,65 @@
 # Front-End for multiplayer video game
-An interpretation of Fort Nerf which was created by Bitwise
+<a name="top"></a>
+An interpretation of Fort Nerf which was created by Bitwise!
 
 # Table of contents
+<details>
+  <summary>Table of Contents</summary>
+  <ol>
+    <li>
+      <a href="#about-the-project">About The Project</a>
+      <ul>
+        <li><a href="#built-with">Built With</a></li>
+        <!-- <li><a href="#wireframe">Wireframe</a></li> -->
+      </ul>
+    </li>
+    <li>
+      <a href="#getting-started">Getting Started</a>
+      <ul>
+        <li><a href="#prerequisites">Prerequisites</a></li>
+        <li><a href="#installation">Installation</a></li>
+      </ul>
+    <li><a href="#contact">Contact</a></li>
+  </ol>
+</details>
 
 # About the Project
 ## About The Project
```

Comment:
i'd prolly jus get rid of one of these 'about the project' lines. preferrably #27 maybe cuz kinda redundant